### PR TITLE
Temp. Remove Generate Audio 1 and Rename Generate Audio 2

### PR DESCRIPTION
**Pull request by ditinagrawal**
Generate Audio 1 is temporarily removed and
Generate Audio 2 is renamed to Generate Audio

### CODE REVIEW

**Comment by AlbertArakelyan** [comment]
@ditinagrawal hey thanks much for contributing :)
one small thing could you please remove `disabled` from the button you added, and also give `id="textToAudio"` to the button you added.

File: index.html [hunk 1]
```diff
@@ -1,71 +1,99 @@
...
+                            <option value="en">English</option>
+                            <option value="ru">Russian</option>
+                        </select>
+                    </div>
+                    <div class="d-flex gap-2 mt-2 align-items-center">
+                        <!-- <button class="btn btn-primary" id="textToAudio">Generate Audio 1</button> -->
+                        <button
+                            class="btn btn-primary"
+                            onclick="convertText()"
+                            disabled
```
Comment (AlbertArakelyan) [inline index.html:81]: Please remove disabled and bring `id="textToAudio"` to this button.

Forgot to mention on the issue :) this button serves as the main feature :)

**Review by AlbertArakelyan — CHANGES_REQUESTED** [review]
One small thing, thank you!

**Comment by AlbertArakelyan** [comment]
nevermind I merged it, will correct, my bad didn't include on the issue ;)

**Comment by ditinagrawal** [comment]
Happy to help 😃

**Comment by AlbertArakelyan** [comment]
@ditinagrawal Could you also please tell your opinion about the app and how useful it can be? 🙂 
Just interesting how was your impression from the project, as I saw other "free" converters they take you through five steps and the end want to your card to be attached ))

Of course I have big work to o for this project in future.

**Pull request merged**